indico-prod/.env

```diff
@@ -0,0 +1,21 @@
+## DB config
+## ===========================================
+PGHOST=indico-postgres
+PGUSER=indico
+PGPASSWORD=indicopass
+PGDATABASE=indico
+
+## Celery
+## ===========================================
+C_FORCE_ROOT=true
+
+## Nginx
+## ===========================================
+NGINX_PORT=8080
+# Should match the 'BASE_URL' in indico.conf
+NGINX_SERVER_NAME=indico-prod-xxx.org
```

Comment:
do we actually need this? i have no problem accessing indico using a different hostname anyway, and if we can get rid of it it'd save quite a bit of complexity (in particular, not having to preprocess the nginx config).

PS: I'd switch the nginx image to `ghcr.io/nginxinc/nginx-unprivileged:stable-alpine` since the docker hub has some nasty pull rate limits for free users